docs/references/api.rst

```diff
@@ -16,3 +16,101 @@ PostgREST exposes three database objects of a schema as resources: tables, views
+      '#function-filters': 'api/stored_procedures.html#table-valued-functions',
+      '#overloaded-functions': 'api/stored_procedures.html#overloaded-functions',
+      // Schemas
+      '#switching-schemas': 'api/schemas.html',
+      // Resource Embedding
+      '#resource-embedding': 'api/resource_embedding.html#resource-embedding',
+      '#many-to-one-relationships': 'api/resource_embedding.html#many-to-one-relationships',
+      '#one-to-many-relationships': 'api/resource_embedding.html#one-to-many-relationships',
+      '#many-to-many-relationships': 'api/resource_embedding.html#many-to-many-relationships',
+      '#one-to-one-relationships': 'api/resource_embedding.html#one-to-one-relationships',
+      '#computed-relationships': 'api/resource_embedding.html#computed-relationships',
+      '#nested-embedding': 'api/resource_embedding.html#nested-embedding',
+      '#embedded-filters': 'api/resource_embedding.html#embedded-filters',
+      '#embedding-with-top-level-filtering': 'api/resource_embedding.html#top-level-filtering',
+      '#embedding-partitioned-tables': 'api/resource_embedding.html#embedding-partitioned-tables',
+      '#embedding-views': 'api/resource_embedding.html#embedding-views',
+      '#embedding-chains-of-views': 'api/resource_embedding.html#embedding-chains-of-views',
+      '#embedding-on-stored-procedures': 'api/resource_embedding.html#embedding-on-stored-procedures',
+      '#embedding-after-insertions-updates-deletions': 'api/resource_embedding.html#embedding-after-insertions-updates-deletions',
+      '#embedding-disambiguation': 'api/resource_embedding.html#embedding-disambiguation',
+      '#target-disambiguation': 'api/resource_embedding.html#target-disambiguation',
+      '#hint-disambiguation': 'api/resource_embedding.html#hint-disambiguation',
+      // OpenAPI
+      '#openapi-support': 'api/openapi.html',
+      // Resource Representation
+      '#response-format': 'api/resource_representation.html#response-format',
+      '#singular-or-plural': 'api/resource_representation.html#singular-or-plural',
+      '#response-formats-for-scalar-responses': 'api/resource_representation.html#scalar-function-response-format',
+      // CORS
+      '#cors': 'api/cors.html',
+      // OPTIONS
+      '#options': 'api/options.html',
+      // URL Grammar
+      '#custom-queries': 'api/url_grammar.html#custom-queries',
+      '#unicode-support': 'api/url_grammar.html#unicode-support',
+      '#table-columns-with-spaces': 'api/url_grammar.html#table-columns-with-spaces',
+      '#reserved-characters': 'api/url_grammar.html#reserved-characters',
+      // Transactions
+      '#immutable-and-stable-functions': 'transactions.html#access-mode',
+      '#http-context': 'transactions.html#transaction-scoped-settings',
+      '#accessing-request-headers-cookies-and-jwt-claims': 'transactions.html#request-headers-cookies-and-jwt-claims',
+      '#legacy-guc-variable-names': 'transactions.html#legacy-guc-variable-names',
```

Comment:
This one slipped me by.. I didn't include this section on the new transactions page

Reply:
Added #626 to address this.